### PR TITLE
Fix tests, add action activation/deactivation

### PR DESCRIPTION
**Pull request by Nicba1010**
**Description**
A long press option on saved devices now pops up a menu to rename, deactivate all actions for device,  activate all actions for device
An activation switch exists in the action detail activity
The action has a dark background when deactivated
Fixes mockito tests

**Issues to solve**
#193 
#194

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/Sensorics.iml" filepath="$PROJECT_DIR$/Sensorics.iml" />
+      <module fileurl="file://$PROJECT_DIR$/Sensorics_.iml" filepath="$PROJECT_DIR$/Sensorics_.iml" />
```
Comment (borna-mdn) [inline .idea/modules.xml]: Why this?

Comment (Nicba1010) [reply .idea/modules.xml]: I've got no ideal, ill check and remove it

Comment (Nicba1010) [reply .idea/modules.xml]: of its in the way I cloned it, i named the directory Sensorics_, ill fix it

Comment (Nicba1010) [reply .idea/modules.xml]: Fixed

**Pull request merged**